htdocs/societe/class/api_thirdparties.class.php

```diff
@@ -114,8 +115,7 @@ function get($id)
 	 * @param   string  $sqlfilters Other criteria to filter answers separated by a comma. Syntax example "(t.nom:like:'TheCompany%') and (t.date_creation:<:'20160101')"
 	 * @return  array               Array of thirdparty objects
 	 */
-    function index($sortfield = "t.rowid", $sortorder = 'ASC', $limit = 100, $page = 0, $mode=0, $sqlfilters = '')
-    {
+	function index($sortfield = "t.rowid", $sortorder = 'ASC', $limit = 100, $page = 0, $mode=0, $sqlfilters = '') {
```

Comment:
please keep opening brace on new line, i have uniformised all the others

Reply:
Thanks @frederic34, sorry about that :confused: 
Must be my IDE ; I have fixed it now :wink: